tests/classes/PluginManager.js

```diff
@@ -765,8 +766,11 @@ describe('PluginManager', () => {
   describe('Plugin/CLI integration', () => {
```

Comment:
this one should stay in this file, because its basically testing that the plugin system works. In general those serverless exec commands are a good fit for integration tests though